src/core/execute.c

```diff
@@ -5221,6 +5221,11 @@ int exec_spawn(Unit *unit,
                         if (r < 0)
                                 return log_unit_error_errno(unit, r, "Failed to create control group '%s': %m", subcgroup_path);
 
+                        r = cg_adjust_threaded(SYSTEMD_CGROUP_CONTROLLER, subcgroup_path);
+                        if (r < 0)
+                                return log_unit_error_errno(unit, r, "Failed to switch control group '%s' to threaded mode: %m",
+                                                            subcgroup_path);
+
```

Comment:
i don#t grok this. i am prety sure we should *never* create threaded cgroups. if people want those, they should do that down the tree, but not where we are affected by it.

Reply:
i am still not getting this. why do we bother with threaded mode? people can do this down the tree, but not up to where it affects us.